pkg/kubectl/cmd/create.go

```diff
@@ -170,6 +187,20 @@ func RunCreate(f cmdutil.Factory, cmd *cobra.Command, out, errOut io.Writer, opt
 	return nil
 }
 
+// decodeUnstructuredObject attempts to convert an unstructured object into a known type
+// If the object fails to be converted or printed, an error is returned.
+func decodeUnstructuredObject(obj runtime.Object) (runtime.Object, bool) {
+	switch obj.(type) {
+	case *runtime.UnstructuredList, *runtime.Unstructured, *runtime.Unknown:
+		if objBytes, err := runtime.Encode(api.Codecs.LegacyCodec(), obj); err == nil {
```

Comment:
@deads2k 
> This looks very, very wrong. What are you trying to do?

I'm attempting to convert to a known type, I used [this implementation in filter.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/resource_filter.go#L65) as an example for this.